<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZwiView</class>
 <widget class="QMainWindow" name="ZwiView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>1061</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1027</width>
      <height>968</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="firstName">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FirstName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lastName">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LastName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="isWer">
       <property name="text">
        <string>follower</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="isWee">
       <property name="text">
        <string>followee</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCommandLinkButton" name="butPrev">
          <property name="text">
           <string>Prev</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="butNext">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="butQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="sb">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZwiView">
    <property name="title">
     <string>ZwiView</string>
    </property>
    <addaction name="actionwers"/>
    <addaction name="actionwees"/>
    <addaction name="separator"/>
    <addaction name="actionauto"/>
    <addaction name="actionreset"/>
    <addaction name="separator"/>
    <addaction name="actionnext"/>
    <addaction name="actionprev"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menuZwiView"/>
  </widget>
  <action name="actionwees">
   <property name="text">
    <string>wees</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionwers">
   <property name="text">
    <string>wers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionauto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>auto</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionprev">
   <property name="text">
    <string>prev</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionreset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
